R/utils.R

```diff
@@ -96,10 +96,47 @@ same_contents <- function(path, contents) {
   identical(text_hash, file_hash)
 }
 
-r_files <- function(path) {
+all_r_files <- function(path) {
   sort_c(dir(file.path(path, "R"), "[.Rr]$", full.names = TRUE))
 }
 
+ignore_files <- function(path, rfiles) {
+    rbuildignore <- file.path(path, ".Rbuildignore")
+     if (file.exists(rbuildignore)) {
```

Comment:
It would make the code simpler to flip the check and return early if there is no `.Rbuildignore`